src/App.js

```diff
@@ -1,24 +1,25 @@
-import logo from './logo.svg';
-import './App.css';
+import { Provider } from "react-redux";
 
-function App() {
+import { AddQuestionForm } from "./addQuestion/index.js";
+import { ListOfQuestions } from "./listOfQuestions/index.js";
+import { Score } from "./score/index.js";
+import { CurrentStreak } from "./currentStreak/index.js";
+
+import "./App.css";
+
+function App({ store }) {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Provider store={store}>
+      <div className="rejection-app-container">
+        <header className={"header"}>
+          <h1 className={"header__title"}>Rejection App</h1>
+        </header>
+        <AddQuestionForm positionClass={"form"} />
+        <ListOfQuestions positionClass={"list"} />
+        <Score positionClass={"score"} />
```

Comment:
`Score` and `CurrentStreak` could be composed into a single component, e.g., `ScoreBoard`.